packages/web/src/components/common/liquidity-enter-amounts/LiquidityEnterAmounts.stories.tsx

```diff
@@ -16,14 +17,15 @@ const Template: ComponentStory<typeof EnterAmounts> = args => (
   <EnterAmounts {...args} />
 );
 
-const token = {
+const token: TokenModel = {
   "chainId": "test3",
   "address": "0x111111111117dC0aa78b770fA6A738034120C302",
   "path": "gno.land/r/demo/1inch",
   "name": "1inch",
   "symbol": "1INCH",
   "decimals": 6,
   "logoURI": "https://assets.coingecko.com/coins/images/13469/thumb/1inch-token.png?1608803028",
+  "type": "grc20",
   "priceId": "1inch",
   "createdAt": "1999-01-01T00:00:01Z"
 };
```

Comment:
can we change this info to grc20 `foo`? please change for all specs and links in this PR.

Reply:
sure. I will change the token used for testing to the foo token of grc20.